components/UpdateProfileForm/steps/MilitaryDetails.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { bool } from 'prop-types';
+import { Field } from 'formik';
+import * as Yup from 'yup';
+import { updateUser } from 'common/constants/api';
+import { validationErrorMessages } from 'common/constants/messages';
+import Input from 'components/Form/Input/Input';
+import Select from 'components/Form/Select/Select';
+import styles from './_steps.css';
+
+class MilitaryDetails extends React.Component {
+  static propTypes = {
+    isSubmitting: bool,
+  };
+
+  static defaultProps = {
+    isSubmitting: false,
+  };
+
+  static validationSchema = Yup.object().shape({
+    // TODO: use when() on these
+    branchOfService: Yup.string().required(validationErrorMessages.required),
+    yearsOfService: Yup.number().required(validationErrorMessages.required),
+    payGrade: Yup.string().required(validationErrorMessages.required),
+  });
+
+  static initialValues = {
+    branchOfService: '',
+    yearsOfService: '',
+    payGrade: '',
+  };
+
+  static submitHandler = async values => {
+    await updateUser(values);
+  };
+
+  render() {
+    const { isSubmitting } = this.props;
+
+    return (
+      <>
+        <h2 className={styles.row}>Military Details</h2>
+
+        <div className={styles.row}>
+          <Field
+            className={styles.fullWidth}
+            name="branchOfService"
+            label="Branch Of Service*"
+            component={Select}
+            options={[
+              {
```

Comment:
Is there a reason this is declared as an option instead of a `placeholder`?

Reply:
Good shout! I can probably just remove this option and leave the default placeholder